### ISSUE TITLE
Medievil fps drops

### ISSUE THREAD

**Issue by QrTa** [hrydgard/ppsspp#15988]
### Game or games this happens in

UCES-00006

### What area of the game

Dan's Crypt if you start spinning at a room

![Screenshot_2022-09-08-11-44-20-531_com google android apps photos](https://user-images.githubusercontent.com/81707167/189090790-c9397dec-9ea6-46eb-8d48-6a3b7269405c.jpg)

Loading screens overally runs at 10-30/60fps 

### Speed seen in PPSSPP

80-100% 50-60fps

### Platform

Android

### Mobile phone model or graphics card

POCO F3 Adreno 650

### PPSSPP version affected

v1.13.1-843-g7450f3ef7

### Graphics backend (3D API)

OpenGL / GLES

### Any other notes or things you've tried

Vulkan, GLES tried got the same result. 

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [ ] Try resetting settings or older versions and include if the issue is related.
- [X] Try changing graphics settings to determine if one causes the slowdown.
- [X] Include logs or screenshots of issue.

**Comment by QrTa** [comment]
Loading screens

https://user-images.githubusercontent.com/81707167/189091747-aaf7762b-0ce1-4914-b59e-31e7de14606d.mp4

Fps drops

https://user-images.githubusercontent.com/81707167/189091999-d875b9af-bcb2-4183-92d0-e1d397309d0d.mp4

**Comment by unknownbrackets** [comment]
If a picture is worth a thousand words and a video is worth ten thousand, a GE frame dump is worth a million words.

-[Unknown]

**Comment by QrTa** [comment]
From the loading screen? Or from that half sec when the fps drops? Then I will try to catch that point.

**Comment by QrTa** [comment]
Tried to catch 3 times, the rexording only started after the fps stabilized so I'm not download them. For the 4th try the debugger freezed at "Recording"  now it's not connects.

**Comment by Gamemulatorer** [comment]
What if you added this game id to ***ForceMax60FPS***?

**Comment by QrTa** [comment]
@Gamemulatorer what is that? How can I add it?

**Comment by Gamemulatorer** [comment]
> @Gamemulatorer what is that? How can I add it?

Create compat.ini file in your psp/system folder.
![Screenshot_2022-09-16-22-31-00-82](https://user-images.githubusercontent.com/37603562/190663548-b2ba1cb3-c33f-4e9b-84ee-a5e4959f1fb7.png)

**Comment by QrTa** [comment]
This one is fixed or at least makes better the loading screens![Screenshot_2022-09-16-16-37-22-653_org.ppsspp.ppsspp.jpg](https://user-images.githubusercontent.com/81707167/190664851-3d42d008-b7bd-40c6-96f6-781235715d37.jpg)

But in the games there are still slowdowns.
![Screenshot_2022-09-16-16-37-03-662_org.ppsspp.ppsspp.jpg](https://user-images.githubusercontent.com/81707167/190664922-f516d8b2-1ee0-48b7-b4d4-dfcdc06daf88.jpg)

**Comment by Gamemulatorer** [comment]
> If a picture is worth a thousand words and a video is worth ten thousand, a GE frame dump is worth a million words.
> 
> -[Unknown]

Here we go...

[UCES00006.ppdmp.zip](https://github.com/hrydgard/ppsspp/files/9585329/UCES00006.ppdmp.zip)

**Comment by Gamemulatorer** [comment]
> There are still slowdowns. ![Screenshot_2022-09-16-16-37-03-662_org.ppsspp.ppsspp.jpg](https://user-images.githubusercontent.com/81707167/190664922-f516d8b2-1ee0-48b7-b4d4-dfcdc06daf88.jpg)

Also on my phone :(
![Screenshot_2022-09-16-23-27-54-31](https://user-images.githubusercontent.com/37603562/190675364-4129e792-1854-437f-972f-15d39c0516e5.png)

**Comment by QrTa** [comment]
@Gamemulatorer I mean mine is running with full speed, but there are places where I stand and starts spinning I got these drops but only for a second or half. This is why I can't catch GE dump from the slowdown.

**Comment by Gamemulatorer** [comment]
> @Gamemulatorer I mean mine is running with full speed, but there are places where I stand and starts spinning I got these drops but only for a second or half. This is why I can't catch GE dump from the slowdown.

Both happens in OGl and VK backend?
Are you using the 120hz screen refresh rate of your phone?

**Comment by QrTa** [comment]
Yep, OGLES and Vulkan. 
I'm using my screen on 60Hz.

**Comment by Gamemulatorer** [comment]
> Yep, OGLES and Vulkan. I'm using my screen on 60Hz.

What happen if you turn off ***hardware transform***?

**Comment by Gamemulatorer** [comment]
No feedback/outdated???

**Comment by hrydgard** [comment]
No, i think there is an actual issue in this game, timing maybe, that should be investigated

**Comment by Gamemulatorer** [comment]
This is the same as https://github.com/hrydgard/ppsspp/issues/12313#issuecomment-903581961 I got stable fps if I overclock it.

**Comment by Gamemulatorer** [comment]
Or this might be improve if ever ***sync to host*** is implemented https://github.com/hrydgard/ppsspp/issues/15081

**Comment by unknownbrackets** [comment]
An FPS drop to 50/60 can't even possibly be helped by that, no.  That's a request to scale game speed to the ratio of your display speed (for example, if your display is 75 Hz, to run at 125% speed, or if your display is 59 Hz, run at 98% speed.)

In this case, the game is (sometimes) outputting less than 59.94 frames per second.  Even if you fast-forwarded emulation constantly to 125% for a 75 Hz display, it would still be outputting 16% less frames than it should, so you'd still get a frame drop.

It would help if someone can check this on a real PSP, just so we're sure there aren't FPS drops in the game itself.  It's probably some timing issue, though, I guess... usually we're faster than we should be, though.

-[Unknown]

**Comment by Gamemulatorer** [comment]
@QrTa what if you set emulated psp cpu clock to 666mhz?

**Comment by NABN00B** [comment]
> It would help if someone can check this on a real PSP, just so we're sure there aren't FPS drops in the game itself. It's probably some timing issue, though, I guess... usually we're faster than we should be, though.

There *is* FPS drop in and around the center room of the level. However, with PPSSPP the game also achieves much higher framerates on the first level in general. See https://github.com/hrydgard/ppsspp/issues/20296#issuecomment-2868848473
